src/App.css

```diff
@@ -46,3 +50,18 @@ select {
 .bg-brown {
   background-color: rgb(var(--clr-brown)) !important;
```

Comment:
not sure about this but using import is not recommendable

Reply:
Removed unwanted !important tag